### DISCUSSION TITLE
Unable to run Stable Diffusion

### DISCUSSION THREAD

**Discussion by vesperialuna** [TheLastBen/fast-stable-diffusion#1506] [General]
Hi TLB, 

I am unable to run the fast SD with these errors:

![image](https://user-images.githubusercontent.com/121345493/215253572-6b1c381d-ae7a-4cb7-8691-73509ae2b72a.png)

I have tried deleting the entire folder, connecting different gdrive, restarting the Colab, clearing cache etc. and same error occurs. Any ideas? 

Thank you in advance.

**Comment by eliasprompt** [comment]
We all have the same issue, just wait to be fixed
